updater/internal/install/install_test.go

```diff
@@ -244,70 +244,6 @@ func TestInstallArtifacts(t *testing.T) {
 			action.NewServiceStopAction(svc),
 		}, actions)
 	})
-
-	t.Run("An artifact exists already as a folder", func(t *testing.T) {
```

Comment:
Just checking we meant to remove this?

Reply:
@cpheps yes. this is the test that i mentioned in slack last night. because now i'm deleting old files before they get copied over (to prevent the weirdness of the collector not being able to run), this test is no longer valid